volunteer.php

```diff
@@ -650,4 +664,62 @@ function volunteer_civicrm_fieldOptions($entity, $field, &$options, $params) {
   if ($entity == "UFJoin" && $field == "entity_table" && $params['context'] == "validate") {
     $options[CRM_Volunteer_DAO_Project::getTableName()] = CRM_Volunteer_DAO_Project::getTableName();
   }
+}
+
+/**
+ * This function fetches the defaults from civicrm settings
+ * And puts them into the appropriate data format to return
+ * to the angular front-end
+ *
+ * @return array
+ * @throws CiviCRM_API3_Exception
+ */
+function volunteer_composeDefaultArray() {
+  $defaults = array();
+  $settings = CRM_Core_BAO_Setting::getItem("volunteer_defaults");
```

Comment:
Since the settings group no longer gains you the convenience factor during retrieval, why not use the same settings group as the other volunteer settings for consistency?